### PR TITLE
Fix biopet 451

### PR DESCRIPTION
**Pull request by akaljuvee**

### CODE REVIEW

File: Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/svcalling/ShivaSvCalling.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import nl.lumc.sasc.biopet.test.shiva.svcallers.SvCaller
+import nl.lumc.sasc.biopet.test.utils.createTempConfig
+
+trait ShivaSvCalling extends Pipeline {
+
+  override def pipelineName = "shivasvcalling"
+
+  def svCallers: List[SvCaller]
+
+  def bamFile: File
```
Comment (ffinfo) [inline Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/svcalling/ShivaSvCalling.scala]: This pipeline allows multiple bam files as input

File: Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/svcalling/ShivaSvCallingSuccess.scala [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package nl.lumc.sasc.biopet.test.shiva.svcalling
+
+import nl.lumc.sasc.biopet.test.PipelineSuccess
+
+trait ShivaSvCallingSuccess extends ShivaSvCalling with PipelineSuccess {
```
Comment (ffinfo) [inline Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/svcalling/ShivaSvCallingSuccess.scala]: This pipeline also has a summary but leave this for now. I will change PipelineSuccess to SummaryPipeline in the new summary branch later. Maybe just add a todo in the code that we don't forget this.

**Review by ffinfo — CHANGES_REQUESTED** [review]
Overall looking great.

Only change that is required is the multiple bam files as input.

**Review by ffinfo — APPROVED** [review]

**Comment by akaljuvee** [comment]
Indeed there should be a test that has multiple samples, changed one test for that. And I chose the wgs1 sample, so now it checks also whether the situation when the tools don't find anything is handled ok.

**Pull request merged**